adserver/tasks.py

```diff
@@ -687,8 +687,9 @@ def calculate_ad_ctrs(days=7):
         total_views = result["total_views"] or 0
         total_clicks = result["total_clicks"] or 0
 
-        ad.sampled_ctr = calculate_ctr(total_clicks, total_views)
-        ad.save()
+        if total_views >= min_views:
+            ad.sampled_ctr = calculate_ctr(total_clicks, total_views)
```

Comment:
Should we set it to `None` if it's below this? That would help with the "ads not being re-evaluated" problem, and at least make sure we always have fresh-ish samples.

Reply:
It's set to not None. We could set it to zero but feels a bit strange to do that.

Reply:
But for ads that got a sampled_ctr set, but then fell below the threshold. They will just have the same, old `sampled_ctr` forever, right?

Reply:
They will until the next time they're processed. However, nothing will deprioritize an ad more than setting it to 0. I think either leave it as-is or we should lower the threshold.